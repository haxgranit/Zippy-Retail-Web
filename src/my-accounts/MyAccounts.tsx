import React from 'react';
import placeholder from './MyAccounts.png';

export default function MyAccounts() {
  return (
    <img src={placeholder} alt="placeholder" style={{ width: '100%' }} />
  );
}
