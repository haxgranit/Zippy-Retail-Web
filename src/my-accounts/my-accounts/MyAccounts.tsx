import React from 'react';
import {
  Alert,
  Card,
  Form,
  InputGroup,
  FormControl,
  Button,
  Nav,
} from 'react-bootstrap';
import CommonHeader from './components/CommonHeader';
import PrintIcon from './icons/print.png';

export default function MyAccounts() {
  return (
    <>
      <CommonHeader title="My Accounts" print />
      <div>
        <div className="d-flex justify-content-between">
          <p>
            Welcome DUANE TOUGH. Your last visit: November 9, 2021 at 4:54 p.m.
            ET
          </p>
          <div className="d-flex">
            <img
              src={PrintIcon}
              alt="placeholder"
              style={{ width: 20, height: 25, marginRight: 10 }}
            />
            <p>Online Security Guarantee</p>
          </div>
        </div>
        <div className="row">
          <div className="col-md-9">
            <Alert variant="primary">
              <div className="d-flex align-items-center">
                <img src={PrintIcon} alt="placeholder" style={{ width: 40, height: 45, marginRight: 10 }} />
                <div>
                  <h3>COVID-19 Support</h3>
                  <p>
                    We
                    {'&apos;'}
                    re commited to supporting you. Find the latest information
                    and available resources.
                  </p>
                  <div className="d-flex justify-content-end">
                    <a href="/home" className="text-danger">
                      Go to COVID-19 Support
                      {' >'}
                    </a>
                  </div>
                </div>
              </div>
            </Alert>
            <Alert variant="primary" dismissible>
              <p style={{ marginTop: 10 }}>
                You
                {'&lsquo;'}
                ve been rewarded with a new offer.
                <a href="/">Check out your offer now before it expires!</a>
              </p>
            </Alert>
            <div className="row">
              <div className="col-md-3 text-center d-flex flex-column align-items-center justify-content-center border-right">
                <p>Deposit Accounts</p>
                <p>$37,139.59</p>
              </div>
              <div className="col-md-3 text-center d-flex flex-column align-items-center justify-content-center border-left">
                <p>+</p>
                <p>Apply for a new product</p>
              </div>
            </div>
            <div className="d-flex justify-content-between">
              <div className="d-flex">
                <p style={{ margin: 'auto' }}>View: </p>
                <Nav>
                  <Nav.Link>List</Nav.Link>
                  <Nav.Link>Pie Charts</Nav.Link>
                </Nav>
              </div>
              <h5>Balance</h5>
            </div>
          </div>
          <div className="col-md-3">
            <Card className="p-3 bg-white">
              <Card.Title>
                Quick Transfer
              </Card.Title>
              <Card.Body>
                <p>Cant find your account in the list?</p>
                <Form.Label>From:</Form.Label>
                <Form.Select>
                  <option>Choose an account</option>
                  <option value="1">One</option>
                  <option value="2">Two</option>
                  <option value="3">Three</option>
                </Form.Select>
                <Form.Label>To:</Form.Label>
                <Form.Select>
                  <option>Choose an account</option>
                  <option value="1">One</option>
                  <option value="2">Two</option>
                  <option value="3">Three</option>
                </Form.Select>
                <Form.Label>Amount:</Form.Label>
                <InputGroup className="mb-3">
                  <InputGroup.Text id="basic-addon1">$</InputGroup.Text>
                  <FormControl
                    placeholder="Username"
                    aria-label="Username"
                    aria-describedby="basic-addon1"
                  />
                </InputGroup>
                <Button variant="danger">Next</Button>
              </Card.Body>
            </Card>
          </div>
        </div>
      </div>
    </>
  );
}
