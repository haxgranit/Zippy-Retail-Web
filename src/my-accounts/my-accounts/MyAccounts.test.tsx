import { render } from '@testing-library/react';

import MyAccounts from './MyAccounts';

describe('MyAccounts Component', () => {
  it('should render MyAccounts Page', () => {
    const { container } = render(<MyAccounts />);
    expect(container).toMatchSnapshot();
  });
});
