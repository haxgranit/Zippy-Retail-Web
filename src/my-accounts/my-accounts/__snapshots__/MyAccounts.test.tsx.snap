// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyAccounts Component should render MyAccounts Page 1`] = `
<div>
  <div
    class="d-flex justify-content-between align-items-center"
  >
    <h3>
      My Accounts
    </h3>
  </div>
  <div
    class="content-wrapper"
  >
    <div
      class="d-flex justify-content-between"
    >
      <p>
        Welcome DUANE TOUGH. Your last visit: November 9, 2021 at 4:54 p.m. ET
      </p>
      <div
        class="d-flex"
      >
        <div
          style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
        >
          P
        </div>
        <p>
          Online Security Guarantee
        </p>
      </div>
    </div>
    <div
      class="row"
    >
      <div
        class="col-md-9"
      >
        <div
          class="fade alert alert-primary show"
          role="alert"
        >
          <div
            class="d-flex align-items-center"
          >
            <div
              style="width: 40px; height: 45px; border: 1px dotted grey; text-align: center;"
            >
              P
            </div>
            <div>
              <h3>
                COVID-19 Support
              </h3>
              <p>
                We're commited to supporting you. Find the latest information and available resources.
              </p>
              <div
                class="d-flex justify-content-end"
              >
                <a
                  class="text-danger"
                  href="/home"
                >
                  Go to COVID-19 Support
                   &gt;
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="fade alert alert-primary alert-dismissible show"
          role="alert"
        >
          <button
            aria-label="Close alert"
            class="btn-close"
            type="button"
          />
          <p
            style="margin-top: 10px;"
          >
            You‘ve been rewarded with a new offer.
            <a
              href="/"
            >
              Check out your offer now before it expires!
            </a>
          </p>
        </div>
        <div
          class="row"
        >
          <div
            class="col-md-3 text-center d-flex flex-column align-items-center justify-content-center border-right"
          >
            <p>
              Deposit Accounts
            </p>
            <p>
              $37,139.59
            </p>
          </div>
          <div
            class="col-md-3 text-center d-flex flex-column align-items-center justify-content-center border-left"
          >
            <p>
              +
            </p>
            <p>
              Apply for a new product
            </p>
          </div>
        </div>
        <div
          class="d-flex justify-content-between"
        >
          <div
            class="d-flex"
          >
            <p
              style="margin: auto;"
            >
              View: 
            </p>
            <div
              class="nav"
            >
              <a
                class="nav-link"
                role="button"
                tabindex="0"
              >
                List
              </a>
              <a
                class="nav-link"
                role="button"
                tabindex="0"
              >
                Pie Charts
              </a>
            </div>
          </div>
          <h5>
            Balance
          </h5>
        </div>
        <div
          class="d-flex align-items-center"
          style="border-top: 3px solid green; border-bottom: 2px solid grey; padding-top: 10px; padding-bottom: 10px; background-color: rgb(242, 243, 242); padding-left: 5px; margin-top: 10px;"
        >
          <span
            style="font-weight: 500;"
          >
            Deposite Accounts
             
            |
             
          </span>
          <a
            class="text-decoration-none"
            href="/"
            style="margin-left: 5px; font-size: 14px; color: rgb(220, 53, 69);"
          >
            Get up to $300 back with ZippyX Global Money Transfer. 
          </a>
        </div>
        <div
          class="row"
        >
          <div
            class="col-md-9"
          >
            <div
              class="d-flex justify-content-between align-items-center"
              style="padding-top: 10px; padding-bottom: 10px;"
            >
              <div
                class="d-flex flex-column"
                style="border-left: 2px solid green; padding-left: 10px;"
              >
                <a
                  href="/"
                  style="color: rgb(220, 53, 69);"
                >
                  Chequing
                </a>
                <span
                  style="font-size: 12px;"
                >
                  01702-81-99639
                </span>
              </div>
              <span>
                $ 2,260.57
              </span>
            </div>
            <div
              class="d-flex justify-content-between align-items-center"
              style="padding-top: 10px; padding-bottom: 10px;"
            >
              <div
                class="d-flex flex-column"
                style="border-left: 2px solid green; padding-left: 10px;"
              >
                <a
                  href="/"
                  style="color: rgb(220, 53, 69);"
                >
                  Chequing
                </a>
                <span
                  style="font-size: 12px;"
                >
                  01702-81-99639
                </span>
              </div>
              <span>
                $ 2,260.57
              </span>
            </div>
            <div
              class="d-flex justify-content-between align-items-center"
              style="padding-top: 10px; padding-bottom: 10px;"
            >
              <div
                class="d-flex flex-column"
                style="border-left: 2px solid green; padding-left: 10px;"
              >
                <a
                  href="/"
                  style="color: rgb(220, 53, 69);"
                >
                  Chequing
                </a>
                <span
                  style="font-size: 12px;"
                >
                  01702-81-99639
                </span>
              </div>
              <span>
                $ 2,260.57
              </span>
            </div>
          </div>
          <div
            class="col-md-3"
          >
            Graph Here
          </div>
        </div>
        <div
          class="d-flex align-items-center"
          style="border-top: 3px solid #B23C54; border-bottom: 2px solid grey; padding-top: 10px; padding-bottom: 10px; background-color: rgb(242, 243, 242); padding-left: 5px; margin-top: 10px;"
        >
          <span
            style="font-weight: 500;"
          >
            Non-Registered Investments
             
            |
             
          </span>
          <a
            class="text-decoration-none"
            href="/"
            style="margin-left: 5px; font-size: 14px; color: rgb(220, 53, 69);"
          />
        </div>
        <div
          style="padding: 10px;"
        >
          <p>
            You don't have any non-registered investments with ZippyX
          </p>
          <a
            href="/"
            style="color: rgb(220, 53, 69); font-size: 14px;"
          >
            Kick-start your investing with ZippyX Investor's Edge
             &gt; 
          </a>
        </div>
        <div
          class="d-flex align-items-center"
          style="border-top: 3px solid #BC6800; border-bottom: 2px solid grey; padding-top: 10px; padding-bottom: 10px; background-color: rgb(242, 243, 242); padding-left: 5px; margin-top: 10px;"
        >
          <span
            style="font-weight: 500;"
          >
            Registered Investments
             
            |
             
          </span>
          <a
            class="text-decoration-none"
            href="/"
            style="margin-left: 5px; font-size: 14px; color: rgb(220, 53, 69);"
          />
        </div>
        <div
          style="padding: 10px;"
        >
          <a
            href="/"
            style="color: rgb(220, 53, 69); font-size: 14px;"
          >
            Grow your money with special interest rates on our TFSA and RRSP savings accounts
             &gt; 
          </a>
        </div>
        <div
          class="d-flex align-items-center"
          style="border-top: 3px solid #0163A6; border-bottom: 2px solid grey; padding-top: 10px; padding-bottom: 10px; background-color: rgb(242, 243, 242); padding-left: 5px; margin-top: 10px;"
        >
          <span
            style="font-weight: 500;"
          >
            Credit
             
            |
             
          </span>
          <a
            class="text-decoration-none"
            href="/"
            style="margin-left: 5px; font-size: 14px; color: rgb(220, 53, 69);"
          />
        </div>
        <div
          style="padding: 10px;"
        >
          <div
            class="d-flex justify-content-end"
          >
            <span
              style="color: grey;"
            >
              Icon
            </span>
            <span>
              Active Card
            </span>
          </div>
          <a
            href="/"
            style="color: rgb(220, 53, 69); font-size: 14px;"
          >
            Join and get up to $1,200 in value with the ZippyX Aventura Visa Infinite Card Conditions apply.
             &gt; 
          </a>
        </div>
        <div
          class="d-flex"
        >
          <div
            class="d-flex flex-column align-items-center"
            style="border-top: 3px solid grey; padding: 25px 10px 40px 10px; flex: 1; margin-left: 5px; background-color: rgb(242, 243, 242);"
          >
            <div
              style="width: 40px; height: 45px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
            >
              P
            </div>
            <p
              style="font-weight: bold; margin-top: 10px;"
            >
              Create a savings goal
            </p>
          </div>
          <div
            class="d-flex flex-column align-items-center"
            style="border-top: 3px solid grey; padding: 25px 10px 40px 10px; flex: 1; margin-left: 5px; background-color: rgb(242, 243, 242);"
          >
            <div
              style="width: 40px; height: 45px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
            >
              P
            </div>
            <span
              style="font-weight: bold; margin-top: 12px;"
            >
              Your net worth:
            </span>
            <span
              style="font-weight: bold;"
            >
              $337,139.59
            </span>
            <div
              class="d-flex flex-column justify-content-start"
              style="width: 100%; margin-top: 15px;"
            >
              <span
                style="font-weight: bold;"
              >
                Assets:
              </span>
              <span>
                $337,139.59
              </span>
              <div
                style="width: 100%; height: 10px; background-color: rgb(94, 92, 153); border-radius: 3px;"
              />
            </div>
            <div
              class="d-flex flex-column justify-content-start"
              style="width: 100%; margin-top: 15px;"
            >
              <span
                style="font-weight: bold;"
              >
                Liabilities:
              </span>
              <span>
                $0.00
              </span>
            </div>
          </div>
          <div
            class="d-flex flex-column align-items-center"
            style="border-top: 3px solid grey; padding: 25px 10px 40px 10px; flex: 1; margin-left: 5px; background-color: rgb(242, 243, 242);"
          >
            <div
              style="width: 40px; height: 45px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
            >
              P
            </div>
            <p
              style="font-weight: bold; margin-top: 10px; text-align: center;"
            >
              Your ZippyX Financial HealthCheck goal is to manage your money better
            </p>
            <span
              style="font-weight: bold;"
            >
              Last completed:
            </span>
            <span>
              September 16, 2021
            </span>
          </div>
        </div>
        <div
          class="d-flex"
          style="padding: 25px; margin-top: 15px; background-color: rgb(242, 243, 242);"
        >
          <div
            style="width: 145px; height: 160px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
          >
            P
          </div>
          <div
            style="padding-left: 15px;"
          >
            <p
              style="font-weight: bold; font-size: 20px; margin-top: 10px;"
            >
              Save up to 10 cents per litre on gas(1)
            </p>
            <p>
              Link and use your ZippyX card with Journie Rewards to save on gas at participating Pioneer, Fas Gas, Ultramar and Chevron gas stations.(2)
            </p>
            <a
              href="/"
              style="color: rgb(220, 53, 69);"
            >
              Link your card
               &gt;
            </a>
          </div>
        </div>
        <div
          class="d-flex"
          style="padding-top: 15px; padding-bottom: 10px; border-top: 1px solid grey; border-bottom: 2px solid grey; margin-top: 10px;"
        >
          <p
            style="color: rgb(90, 95, 97);"
          >
            ZippyX is a member of Canada Deposit Insurance Corporation (CDIC)
          </p>
          <button
            class="d-flex align-items-center btn btn-primary"
            style="border-radius: 50px; background-color: rgb(100, 41, 109); height: 50px;"
            type="button"
          >
            <div
              style="width: 20px; border: 1px dotted grey; text-align: center; margin-right: 5px;"
            >
              P
            </div>
            <div
              class="d-flex flex-column align-items-start justify-content-start"
            >
              <span
                style="font-size: 13px;"
              >
                DEPOSIT
              </span>
              <span
                style="font-size: 13px;"
              >
                PROTETION
              </span>
            </div>
          </button>
        </div>
        <div
          style="margin-top: 10px;"
        >
          <p
            style="font-weight: bold;"
          >
            Note:
          </p>
          <p
            style="color: rgb(90, 95, 97);"
          >
            "My Accounts" includes the online accounts you hold with ZippyX and its affilliates. For help, please contact the ZippyX Online Banking support line at 1-888-872-2422
          </p>
        </div>
        <div
          style="padding: 10px; background-color: rgb(242, 243, 242); border-bottom: 1px solid grey;"
        >
          <span
            style="color: rgb(220, 53, 69);"
          >
            +
          </span>
          <span
            style="font-weight: 500;"
          >
            Trademarks and Disclaimers
          </span>
        </div>
      </div>
      <div
        class="col-md-3"
      >
        <div
          class="p-3 bg-white card"
        >
          <div
            class="card-title h5"
          >
            Quick Transfer
          </div>
          <div
            class="card-body"
          >
            <p>
              Cant find your account in the list?
            </p>
            <label
              class="form-label"
            >
              From:
            </label>
            <select
              class="form-select"
            >
              <option>
                Choose an account
              </option>
              <option
                value="1"
              >
                One
              </option>
              <option
                value="2"
              >
                Two
              </option>
              <option
                value="3"
              >
                Three
              </option>
            </select>
            <label
              class="form-label"
            >
              To:
            </label>
            <select
              class="form-select"
            >
              <option>
                Choose an account
              </option>
              <option
                value="1"
              >
                One
              </option>
              <option
                value="2"
              >
                Two
              </option>
              <option
                value="3"
              >
                Three
              </option>
            </select>
            <label
              class="form-label"
            >
              Amount:
            </label>
            <div
              class="mb-3 input-group"
            >
              <span
                class="input-group-text"
                id="basic-addon1"
              >
                $
              </span>
              <input
                aria-describedby="basic-addon1"
                aria-label="Amount"
                class="form-control"
                placeholder="Amount"
              />
            </div>
            <button
              class="zippy-btn btn btn-primary"
              type="button"
            >
              Next
            </button>
          </div>
        </div>
        <div
          class="bg-white card"
          style="margin-top: 10px;"
        >
          <div
            class="card-body"
          >
            <p>
              My Profile
            </p>
            <div
              style="border-bottom: 1px solid grey;"
            >
              <div
                class="d-flex justify-content-between"
              >
                <p>
                  <span
                    style="color: rgb(220, 53, 69);"
                  >
                    -
                  </span>
                  <span
                    style="font-weight: bold; font-size: 14px;"
                  >
                    Home Address
                  </span>
                </p>
                <span>
                  Edit
                </span>
              </div>
              <div
                style="padding-left: 10px;"
              >
                <p
                  style="font-size: 14px;"
                >
                  717-257 ADELAIDE ST W TORONTO, ON M5H 4E7
                </p>
              </div>
            </div>
            <div>
              <div
                class="d-flex justify-content-between"
              >
                <p>
                  <span
                    style="color: rgb(220, 53, 69);"
                  >
                    -
                  </span>
                  <span
                    style="font-weight: bold; font-size: 14px;"
                  >
                    Contact Details
                  </span>
                </p>
                <span>
                  Edit
                </span>
              </div>
              <div
                style="padding-left: 10px;"
              >
                <span
                  style="font-weight: bold; font-size: 14px;"
                >
                  Email
                </span>
                <p
                  style="font-size: 14px;"
                >
                  dtough@hotmail.com
                </p>
                <span
                  style="font-weight: bold; font-size: 14px;"
                >
                  Home
                </span>
                <p
                  style="font-size: 14px;"
                >
                  410-553-1372
                </p>
                <span
                  style="font-weight: bold; font-size: 14px;"
                >
                  Mobile
                </span>
                <p
                  style="font-size: 14px;"
                >
                  410-553-1372
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          class="bg-white card"
          style="margin-top: 10px;"
        >
          <div
            class="d-flex flex-column card-body"
          >
            <p>
              Quick Links
            </p>
            <a
              class="text-decoration-none text-black"
              href="/"
              style="font-size: 12px; margin-top: 5px; margin-bottom: 5px;"
            >
              Stop payments
              <span
                style="color: rgb(220, 53, 69); margin-left: 5px;"
              >
                &gt;
              </span>
            </a>
            <a
              class="text-decoration-none text-black"
              href="/"
              style="font-size: 12px; margin-top: 5px; margin-bottom: 5px;"
            >
              Change your password
              <span
                style="color: rgb(220, 53, 69); margin-left: 5px;"
              >
                &gt;
              </span>
            </a>
            <a
              class="text-decoration-none text-black"
              href="/"
              style="font-size: 12px; margin-top: 5px; margin-bottom: 5px;"
            >
              Void Cheque / Direct Deposit Form (PDF, 510KB)
              <span
                style="color: rgb(220, 53, 69); margin-left: 5px;"
              >
                &gt;
              </span>
            </a>
            <a
              class="text-decoration-none text-black"
              href="/"
              style="font-size: 12px; margin-top: 5px; margin-bottom: 5px;"
            >
              Sign up for and view your eStatements
              <span
                style="color: rgb(220, 53, 69); margin-left: 5px;"
              >
                &gt;
              </span>
            </a>
            <a
              class="text-decoration-none text-black"
              href="/"
              style="font-size: 12px; margin-top: 5px; margin-bottom: 5px;"
            >
              Go to ZippyX Rewards
              <span
                style="color: rgb(220, 53, 69); margin-left: 5px;"
              >
                &gt;
              </span>
            </a>
            <a
              class="text-decoration-none text-black"
              href="/"
              style="font-size: 12px; margin-top: 5px; margin-bottom: 5px;"
            >
              Apply for a credit limit increase
              <span
                style="color: rgb(220, 53, 69); margin-left: 5px;"
              >
                &gt;
              </span>
            </a>
            <a
              class="text-decoration-none text-black"
              href="/"
              style="font-size: 12px; margin-top: 5px; margin-bottom: 5px;"
            >
              Register for FREE SPC (Student and Youth Clients)
              <span
                style="color: rgb(220, 53, 69); margin-left: 5px;"
              >
                &gt;
              </span>
            </a>
            <a
              class="text-decoration-none text-black"
              href="/"
              style="font-size: 12px; margin-top: 5px; margin-bottom: 5px;"
            >
              Link my cards to Journie Rewards
              <span
                style="color: rgb(220, 53, 69); margin-left: 5px;"
              >
                &gt;
              </span>
            </a>
            <div
              class="d-flex flex-column justify-content-center align-items-center"
            >
              <button
                class="btn btn-outline-danger"
                style="font-size: 13px;"
                type="button"
              >
                Show more
              </button>
              <a
                class="text-decoration-none text-black"
                href="/"
                style="font-size: 13px; margin-top: 5px;"
              >
                Show all
              </a>
            </div>
          </div>
        </div>
        <div
          class="card"
          style="margin-top: 10px; background-color: rgb(242, 243, 242);"
        >
          <div
            style="width: 195px; height: 145px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
          >
            P
          </div>
          <div
            class="card-body"
          >
            <p
              class="card-text"
            >
              Travelling out of province or the country?
            </p>
            <p
              style="font-size: 13px;"
            >
              Make sure you're protected. Consider ZippyX Travel Medical Insurancce
            </p>
            <a
              href="/"
              style="color: rgb(220, 53, 69);"
            >
              Learn More
               &gt; 
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="footer-content-wrapper"
  >
    <div
      class="quick-links container"
    >
      <div
        class="row"
      >
        <h3
          class="col-sm-12"
        >
          Quick Links
        </h3>
      </div>
      <div
        class="row"
      >
        <div
          class="col-sm-2"
        >
          <ul>
            <li>
              <a
                class="text-decoration-none"
                href="/"
              >
                Home
              </a>
            </li>
            <li>
              Please Review Our Legal Stuff by visiting our
               
              <a
                class="text-decoration-none"
                href="/legal"
              >
                Privacy Policy | TOU | EULA | CardHolder T&C Pages
              </a>
            </li>
          </ul>
        </div>
        <div
          class="col-sm-2"
        >
          8465 W Sahara Ave. Las Vegas, NEVADA
          <ul>
            <li>
              <a
                href="tel:+17026231876"
              >
                1.702.623.1876
              </a>
            </li>
            <li>
              <a
                href="mailto:contact@zippy.cash"
              >
                contact@zippy.cash
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col-md-12 col-sm-12 col-xs-12 text-left"
        >
          © 2021 Zippy Cash LLC  All rights reserved.
          <br />
          <br />
          The Program Manager for the Zippy Cash® Prepaid Visa Debit Card is PayQuicker, LLC
          <br />
          Zippy Cash® is a registered trademark of Zippy Cash, LLC
          <br />
          The Zippy Cash Prepaid Visa Debit Card is issued by Metropolitan Commercial Bank (Member FDIC) pursuant to a license from Visa U.S.A. Inc. “Metropolitan Commercial Bank” and “Metropolitan” are registered trademarks of Metropolitan Commercial Bank ©2014. BY USING THIS CARD YOU AGREE TO THE TERMS AND CONDITIONS OF THE CARDHOLDER AGREEMENT AND FEE SCHEDULE, IF ANY. All trademarks and service marks are the property of their respective owners.
          <br />
          In compliance with the USA PATRIOT Act* and other laws, anyone who is a US resident over 18 with a valid Social Security number and a verifiable physical US address (no PO boxes or APO addresses) can apply for an Zippy Cash Card. This information must be provided, along with other information pertinent to receiving approval, when applying for a Zippy Cash Card. Currently, we do not issue the Zippy Cash Card to residents of Puerto Rico, Guam, the Virgin Islands, American Samoa, or other US territories.
          <br />
          *Important information for opening a card account: To help the federal government fight the funding of terrorism and money laundering activities, the USA PATRIOT Act requires all financial institutions and their third parties to obtain, verify and record information that identifies each person who opens a card account. What this means for you: When you open a card account, we will ask for your name, address, date of birth, social security number or country identification number, and other information that will allows us to identify you. We may also request a copy of your driver’s license or other documentation bearing your photo as verification of your identity.
          <br />
          Apple and the Apple logo are trademarks of Apple Inc., registered in the U.S. and other countries. App Store is a service mark of Apple Inc. Google Play is a trademark of Google Inc. Data rates may apply. Samsung Pay is a trademark of Samsung Inc.
        </div>
      </div>
    </div>
    <footer
      class="container mb-3"
    >
      <div
        class="d-none d-md-block"
      >
        <div
          class="d-flex justify-content-between border-bottom"
        />
        <div
          class="d-flex justify-content-between border-top"
        >
          <div
            class=""
          >
            <div
              class="footer-link mt-1"
            >
              <a
                class="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              class="footer-link mt-1"
            >
              <a
                class="text-decoration-none"
                href="/"
              >
                Lore
              </a>
            </div>
            <div
              class="footer-link mt-1"
            >
              <a
                class="text-decoration-none"
                href="/"
              >
                Ipsum
              </a>
            </div>
            <div
              class="footer-link mt-1"
            >
              <a
                class="text-decoration-none"
                href="/"
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
          <div
            class="copyright mt-1"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
      <div
        class="d-block d-md-none"
      >
        <div
          class="d-flex border-bottom"
        >
          <div
            class="d-flex flex-column me-2"
          >
            <div
              class="footer-link mt-1"
            >
              <a
                class="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              class="footer-link mb-1"
            >
              <a
                class="text-decoration-none"
                href="/"
              >
                Lore
              </a>
            </div>
            <div
              class="footer-link mb-1"
            >
              <a
                class="text-decoration-none"
                href="/"
              >
                Ipsum
              </a>
            </div>
            <div
              class="footer-link mb-1"
            >
              <a
                class="text-decoration-none"
                href="/"
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
        </div>
        <div
          class="d-flex justify-content-end border-top"
        >
          <div
            class="copyright mt-2 text-end small"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
